src/Swarm/TUI/Launch/Controller.hs

```diff
@@ -136,7 +137,24 @@ handleLaunchOptionsEvent siPair = \case
         activateFocusedControl x
       _ -> return ()
   CharKey ' ' -> activateControl
-  Key V.KEnter -> activateControl
+  Key V.KEnter -> do
+    activateControl
+    fr <- use $ uiState . uiLaunchConfig . controls . scenarioConfigFocusRing
```

Comment:
@kostmo should I just factor out a "cacheAutoplay" function?

I am not sure how the `cacheValidated<X>Input` functions are supposed to work, so any tips would be appreciated.